app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationEntity.kt

```diff
@@ -72,6 +74,7 @@ data class ConversationAccountEntity(
     }
 }
 
+/** A previously-displayed status which has been saved to disk. */
```

Comment:
@connyduck In the Mastodon API the `Converstation` entity contains a regular `Status` entity.

Why is it a different type here? And can that reasoning please be added to the comment.